src/components/commerce/CommerceSchedule.js

```diff
@@ -116,6 +136,10 @@ class CommerceSchedule extends Component {
     this.props.onScheduleValueChange({ prop: 'slots', value: newSlots });
   };
 
+  onCourtTypesFilterValueChange = selectedCourtTypes => {
+    this.setState({ selectedCourtTypes }, this.reservationsOnSlots);
```

Comment:
No hace falta que `this.reservationsOnSlots` se ejecute con arrow function?

Reply:
Si recibiera parametros si. pero ahi en ese caso es como cuando pasas una funcion a un onPress